xla/backends/gpu/collectives/nccl_collectives.cc

```diff
@@ -114,6 +115,16 @@ static ncclConfig_t AsNcclConfig(const GpuCollectives::Config& config) {
   if (config.max_nchannels > 0) {
     VLOG(1) << "Maximum number of channels is set to: " << comm_config.maxCTAs;
     comm_config.maxCTAs = config.max_nchannels;
+  } else if (stream_executor->GetDeviceDescription()
+                 .cuda_compute_capability()
+                 .IsBlackwell()) {
+#if (NCCL_VERSION_CODE >= 22800)
```

Comment:
As far as I understand this doesn't need to be a preprocessor `#if`. Can you just make this a normal C++ if-branch. That makes testing this code easier.